configs/terminate_http3_connect_udp.yaml

```diff
@@ -0,0 +1,77 @@
+# This configuration terminates a CONNECT-UDP request and sends UDP payloads directly over UDP to the target.
+admin:
+  address:
```

Comment:
is admin required ?

its generally better not to add unless required

Reply:
same elsewhere

Reply:
Removed the admin configs.